src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -65,6 +65,10 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Resources.resx">
+      <GenerateSource>true</GenerateSource>
+      <Generator>MSBuild:_GenerateResxSource</Generator>
+    </EmbeddedResource>
```

Comment:
uh oh

Reply:
ya kinda odd. I added an additional RESX to match what the EditorFactory docs recommend. I wonder if Arcade will take care of this for us even for the RazorExtension project. I'll play around with it to see if it does.

Reply:
Yup, need it